leavestracker/apps/leaves/constants.py

```diff
@@ -0,0 +1,8 @@
+DEFAULT_ERROR="Something went wrong!"
+LEAVE_SUCCESS_MSG="your leave has been added successfully"
+DUPLICATE_LEAVE_RAISE_ERROR="Entry Exist"

```

Comment:
Can we add detailed reason i.e instead of entry exist we can have Leave already exists.